### PR TITLE
test(integration): use common values.yaml for resources, access keys/IDs and receiver-mock URL

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: tests/integration/values/values_common.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+sumologic:
```
Comment (swiatekm-sumo) [inline tests/integration/values/values_common.yaml:1]: Can you also add a short comment here, explaining what this is for?

File: tests/integration/internal/stepfuncs/debug.go [hunk 2]
```diff
@@ -32,6 +32,8 @@ func PrintClusterStateOpt(force ...bool) features.Func {
 			)
 
 			k8s.RunKubectl(t, ctxopts.KubectlOptions(ctx), "get", "all")
+			k8s.RunKubectl(t, ctxopts.KubectlOptions(ctx), "get", "pvc")
+			k8s.RunKubectl(t, ctxopts.KubectlOptions(ctx), "get", "pv")
```
Comment (swiatekm-sumo) [inline tests/integration/internal/stepfuncs/debug.go:36]: Can you move this to a separate commit?

**Review by swiatekm-sumo — CHANGES_REQUESTED** [review]
Good idea, left some minor comments.

File: tests/integration/internal/stepfuncs/debug.go (hunk 2 above)
Comment (pmalek-sumo) [reply tests/integration/internal/stepfuncs/debug.go]: Done

File: tests/integration/values/values_common.yaml (hunk 1 above)
Comment (pmalek-sumo) [reply tests/integration/values/values_common.yaml:8]: Done

**Review by swiatekm-sumo — APPROVED** [review]
